### PR TITLE
Add screenshot examples for feeding america; Support long url

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: static/js/shared/child_places.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ export function ChildPlaces(props: ChildPlacesPropType): JSX.Element {
   return (
     <div id="child-places">
       <span id="child-place-head">Places in {props.parentPlace.name}</span>
-      {Object.keys(props.childPlaces).map((placeType) => (
+      {[...Object.keys(this.props.childPlaces)].sort().map((placeType) => (
```
Comment (chejennifer) [inline static/js/shared/child_places.tsx:43]: curious why we need to make a new array before sorting? I think you can just do `Object.keys(this.props.childPlaces).sort()`?

**Review by chejennifer — APPROVED** [review]
nice!

File: static/js/shared/child_places.tsx (hunk 1 above)
Comment (shifucun) [reply static/js/shared/child_places.tsx]: copied from another place, simplified now.

**Review by shifucun — COMMENTED** [review]
thanks for review!

**Pull request merged**